modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase

```diff
@@ -1096,4 +1096,29 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-171627. Verify that that on Target Market, just 1 ~ 3 options can be selected "
+	@priority = 4
+	test ViewTargetMarketOptionsError {
+		property solutions.acceptance = "true";
+		property test.name.skip.portal.instance = "PRMMDFRequest#OnlyManagersCanSeeStateManager";
+
+		task ("Given I go to add a new MDF Request") {
+			PRMMarketing.gotoAddNewMDF();
+		}
+
+		task ("When the user selects 4 different Target Market options") {
+			PRMMDFRequest.fillGoalsPage(targetMarketOptions = "Education,Energy,Automotive,Agriculture");
+		}
+
+		task ("Then the error message that exceeded the limit should appear") {
+			PRMMDFRequest.verifyErrorMessege(errorMessage = "You have exceed the choose limit");
+		}
+
+		task ("But when the page is updated and adding only 3 tickets, the error message is not on the screen") {
```

Comment:
Hi @pedroamarall, I think that we can write these steps better. 
We can separate this into two different tasks:
The first one could be:
```
task ("But when I unselect one option, leaving only 3 options selected") {
      PRMMDFRequest.fillGoalsPage(targetMarketOptions = "Education");
}
```
And the second one could be:
```
task ("then the error will no longer be visible") {
      PRMMDFRequest.verifyErrorNotMessege(errorMessage = "You have exceed the choose limit");
}
```

What do you think?

Reply:
Done, thanks!